docs/install/install-from-package/install-agent.rst

```diff
@@ -87,7 +87,7 @@ would be:
    scratch-size = "1G"
 
    [watcher]
-   service-addr = { host = "bai-m1", port = 6009 }
+   service-addr = { host = "127.0.0.1"", port = 6009 }
```

Comment:
Agent Watcher exposes API endpoints to Manager, therefore binding to 127.0.0.1 is not advisable since Manager and Agent can be installed in different hosts. Instead, bind to a host interface that the Manager can access, or revert to using the host alias.

Reply:
With the all respect, I don't understand at all. This setting is used when launching aiohttp's TCPSite, and if you enter the Manager's hostname in the Agent machine, the server will not run.
Also, install from source document and sample tomls say to set to AGENT IP or 127.0.0.1.

Reply:
I did not say to set this as the "Manager's hostname." Instead, I mentioned setting it to the (Agent's) host interface that the Manager can access or connect to. As I said, if the Manager and Agent are installed on separate servers, the Manager cannot send requests to Agent Wather which binds to 127.0.0.1.

Reply:
@adrysn reverted to use the host alias.